### PR TITLE
Feat/remove abi encode

### LINKED ISSUE #26: feat: leverage `AdjustPoolTxDataQueued` event details in test suite
this will also allow us to get rid of the confusing `execPayload_` pattern in `adjustPool`

### PR DESCRIPTION
**Pull request by petrovska-petro**
closes #26

### CODE REVIEW

File: src/RoboSaverVirtualModule.sol [hunk 1]
```diff
@@ -222,7 +221,9 @@ contract RoboSaverVirtualModule {
             abi.encodeWithSelector(IVault.exitPool.selector, BPT_STEUR_EURE_POOL_ID, _card, payable(_card), request_);
         delayModule.execTransactionFromModule(address(BALANCER_VAULT), 0, payload, 0);
 
-        emit AdjustPoolTxDataQueued(address(BALANCER_VAULT), payload);
+        /// @dev event is leverage by off-chain service to execute the transaction
```
Comment (gosuto-inzasheru) [inline src/RoboSaverVirtualModule.sol:224]: i think it is better to make this part of the natspec where the event is defined

File: src/RoboSaverVirtualModule.sol [hunk 2]
```diff
@@ -266,7 +267,9 @@ contract RoboSaverVirtualModule {
         /// @dev Last argument `1` stands for `OperationType.DelegateCall`
         delayModule.execTransactionFromModule(MULTICALL3, 0, multicallPayload, 1);
 
-        emit AdjustPoolTxDataQueued(MULTICALL3, multicallPayload);
+        /// @dev event is leverage by off-chain service to execute the transaction
```
Comment (gosuto-inzasheru) [inline src/RoboSaverVirtualModule.sol:270]: idem

**Comment by gosuto-inzasheru** [comment]
very nice to make `adjustPool` much more intuitive

File: src/RoboSaverVirtualModule.sol (hunk 2 above)
Comment (petrovska-petro) [reply src/RoboSaverVirtualModule.sol]: 2b2553a60543fa7c1fad034959f8f012a8243e5b

File: src/RoboSaverVirtualModule.sol (hunk 1 above)
Comment (petrovska-petro) [reply src/RoboSaverVirtualModule.sol]: 2b2553a60543fa7c1fad034959f8f012a8243e5b

**Review by gosuto-inzasheru — APPROVED** [review]

**Pull request merged**